app/src/main/java/de/westnordost/streetcomplete/quests/QuestModule.kt

```diff
@@ -231,6 +232,7 @@ import javax.inject.Singleton
         AddPostboxRef(),
         AddWheelchairAccessToiletsPart(),
         AddBoardType(),
+        AddPoliceType(),
```

Comment:
Is anyone using this data already?

If adding `name` tag will stay - then it can be moved higher, as `name` tag for police station is widely used (Default Map style for start)

Reply:
As far as I know no one at the moment, but I'd love to see  some providers using it 😁